### PR TITLE
Update unit tests

### PR DESCRIPTION
**Pull request by SpiderDan98**
- Update unit tests

### CODE REVIEW

File: tests/FondOfSpryker/Zed/PersistentCart/Business/Model/QuoteDeleterTest.php [hunk 1]
```diff
@@ -1,157 +0,0 @@
-<?php
```
Comment (daniel-rose) [inline tests/FondOfSpryker/Zed/PersistentCart/Business/Model/QuoteDeleterTest.php]: Why?

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: tests/FondOfSpryker/Zed/PersistentCart/Business/Model/QuoteDeleterTest.php (hunk 1 above)
Comment (SpiderDan98) [reply tests/FondOfSpryker/Zed/PersistentCart/Business/Model/QuoteDeleterTest.php]: Could not solve the error in the partner shop

**Review by daniel-rose — APPROVED** [review]
Don't remove a test. Fix the error.

**Review by daniel-rose — CHANGES_REQUESTED** [review]
Fix the error. Don't remove a test.

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**